### ISSUE TITLE
Not getting the expected output. [Differs in "base_SdssShape_flag" column]

### ISSUE THREAD

**Issue by victor-ren** [lsst/lsst_dm_stack_demo#2]
Compared using ```bin/compare``` script, my demo output file differs from the expected output only in column  "base_SdssShape_flag".
```
Failed (3996 of 4699 flags do not match) in column base_SdssShape_flag.
```
All the other data are the same as expected output or within the desired tolerance.

The LSST stack version I'm using is 10.1. I have tested on both Ubuntu 14.04 and OS X 10.10 Yosemite. The result is the same. I wonder if this is due to different version of packages that were used to generate reference output.

**Comment by laurenam** [comment]
Hi Wei,

This issue has been resolved in DM-2875.  Essentially, a bug was causing the 
base_SdssShape_flag to be set when really it shouldn’t have.  The fix was 
merged to meas_base on June 3, so won’t be in your stack.  You seem to 
be running a current version of the demo repo (I added that error message
as part of the above ticket), so your run is using the buggy flag setting, but
it’s being compared against the “repaired” expected files…so they don’t
match.

cheers,
Lauren
